x-pack/plugins/security_solution/common/detection_engine/get_query_filter.ts

```diff
@@ -121,6 +121,20 @@ export const buildEqlSearchRequest = (
         },
       },
       event_category_field: eventCategoryOverride,
+      fields: [
+        {
+          field: '*',
+          include_unmapped: true,
+        },
+        {
+          field: '@timestamp',
+          // BUG: We have to format @timestamp until this bug is fixed with epoch_millis
+          // https://github.com/elastic/elasticsearch/issues/74582
+          // TODO: Remove epoch and use the same techniques from x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_events_query.ts
+          // where we format both the timestamp and any overrides as ISO8601
+          format: 'epoch_millis',
```

Comment:
Nice catch on the EQL side with this one -- thanks for reporting to the ES side and documenting here 👍